src/heap/src/Common.h

```diff
@@ -9,30 +9,49 @@
 extern "C" {
 #endif
 
-#pragma once
-#pragma pack(push, include, 1) // for byte alignment
-
 /**
  * 'Magic' - aka a sentinel which will be added as a guard band for each allocation in debug mode
  */
 #define ALLOCATION_HEADER_MAGIC "__HEADER_MAGIC_GUARD_BAND__"
 #define ALLOCATION_FOOTER_MAGIC "__FOOTER_MAGIC_GUARD_BAND__"
 
+// Ensure it's more than the actual header and footer size
```

Comment:
It would be clearer to say more than size of ALLOCATION_HEADER_MAGIC and ALLOCATION_FOOTER_MAGIC

Reply:
These are called guard bands in allocators